_topic_map.yml

```diff
@@ -616,6 +616,9 @@ Topics:
   - Name: Sysctls
     File: sysctls
     Distros: openshift-origin,openshift-enterprise
+  - Name: Encrypting Data at Rest
+    File: encrypting_data
+    Distros: openshift-origin,openshift-enterprise,openshift-dedicated
```

Comment:
Is this list correct? Most of the entries in this section don't include openshift-dedicated and I don't know why.

Reply:
@mburke5678 Do you know something about ^^^ ?

Reply:
Per @adellape Whether a doc goes in Dedicated "it usually depends on whether the "dedicated-cluster-admin" role can operate on whatever the topic is talking about. see: https://docs.openshift.com/dedicated/admin_guide/index.html"
Also, "if it does require modifying the master_config or node_config files, probably not relevant to OSD."
Your file required editing master config, so it would not go in Dedicated.

Reply:
Ok, thanks! I've pushed an update.